Linkbrary/src/js/formValidation.js

```diff
@@ -0,0 +1,12 @@
+import isValidEmail from "./emailValidation";
+import isValidPassword from "./passwordValidation";
+
+export function isValidInput(emailInput, passwordInput) {
+  return isValidEmail(emailInput) && isValidPassword(passwordInput);
+}
+
+export function isCorrectInput(emailInput, passwordInput) {
+  const email = "test@codeit.com";
+  const password = "codeit101";
```

Comment:
아예 상수로 분리할 수도 있을 것 같아요

```suggestion
  const EMAIL = "test@codeit.com";
  const PASSWORD = "codeit101";
```

로 해서 파일 상단에 두거나 별도 파일로 분리

Reply:
수정했습니다